apps/galaxy/src/services/signup/useSignUp.ts

```diff
@@ -0,0 +1,102 @@
+import axios from "../utils/axios";
+import { useState } from "react";
+import { useRouter } from "next/router";
+import { useToast } from "@cc/ui-chakra";
+import { AxiosError } from "axios";
+import { FormData } from "@src/modules/auth/SignUpForm/SignUpForm";
+
+export default function useSignUp() {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isEmailExists, setIsEmailExists] = useState(true);
+  const toast = useToast();
+  const router = useRouter();
+  const handleSignUpSubmit = async (data: FormData) => {
+    setIsLoading(true);
+    try {
+      const response = await axios.post("/franchisee/signup", {
+        franchiseePostDto: {
```

Comment:
Although I know this is the back-end, the naming of this one is very strange, it should be called xxxxpaylod or xxxinput is better

Reply:
Reasonable suggestions, I have given feedback to teammates who did this part.